apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/ErrorReporting/ReportEventsTo.cs

```diff
@@ -43,7 +43,7 @@ public sealed class ReportEventsTo
         internal static readonly MonitoredResource GlobalResource = new MonitoredResource { Type = "global" };
 
         /// <summary>The location to send error events to.</summary>
-        public ReportEventsToLocation ReportEventsToLocation { get; private set; }
+        public EventTarget Location { get; private set; }
```

Comment:
You may want to change this property name to `Target` to match the new naming, but I don't feel too strongly about it.